index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header class="main-nav">
+            <nav>
+                <div class="left-bar">
+                    <p class="time">9:41</p>
+                    <p class="name">Oscar Byiringiro</p>
+                </div>
+                <div class="right-bar">
+                    <div class="top-icons">
+                        <img src="./images/Cellular Connection.png">
+                        <img src="./images/Wifi.png">
+                        <img src="./images/Battery.png">
+                                    
+                                
+                    </div>
+                    <div class="bottom-icons">
+                        <img src="./images/Union.png">
+                            
+                    </div>
+                    
+                        
+                </div>
+            </nav>
+        </header>
+        <section class="main-body">
+            <div class="container">
+                <div class="intro">
+                    <h1>Hey There</h1>
+                    <h1>I'm Oscar</h1>
+                </div>
+                <div class="title">
+                    <p>I'am a software developer</p>
+                </div>
+                <div class="description">
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.</p>
+                </div>
+                <div class="logo">
+                    <img src="./images/twitter.png">
+                    <div class="linkedin">
+                        <img src="./images/Group-1.png">
+                            
+                        <img src="./images/Group.png">
```

Comment:
- [ ] Your logo and the headline text are not aligned vertically. Please see the Figma design and your design. 
Kindly fix this.

Figma design | Your design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/212724607-4cfed79f-22a8-48d8-a9e8-b8de0678345b.png) | ![image](https://user-images.githubusercontent.com/84156714/212724739-65b3f12b-534b-41b6-9ce1-2529108322a9.png)